File: ios/MidObjcSample/MidObjcSample/ResultViewController.m
//
//  ResultViewController.m
//  MidObjcSample
//
//  Created by Muhammad Fauzi Masykur on 25/03/21.
//  Copyright © 2021 Muhammad Fauzi Masykur. All rights reserved.
//

#import "ResultViewController.h"

@interface ResultViewController ()

@end

@implementation ResultViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    [self.paymentStatusLabel setText:self.paymentStatus];
}
@end

File: ios/MidSwiftSample1/MidSwiftSample1/ResultViewController.swift
//
//  ResultViewController.swift
//  MidSwiftSample1
//
//  Created by Muhammad Fauzi Masykur on 26/08/20.
//  Copyright © 2020 Muhammad Fauzi Masykur. All rights reserved.
//

import UIKit

class ResultViewController: UIViewController {

    @IBOutlet weak var paymentStatusLabel: UILabel!
    var paymentStatus = ""
    override func viewDidLoad() {
        super.viewDidLoad()

        // Do any additional setup after loading the view.
        self.paymentStatusLabel.text = self.paymentStatus
    }
    

    /*
    // MARK: - Navigation

    // In a storyboard-based application, you will often want to do a little preparation before navigation
    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
        // Get the new view controller using segue.destination.
        // Pass the selected object to the new view controller.
    }
    */

}
